### ISSUE TITLE
Enable dual control in keyboardController app

### ISSUE THREAD

**Issue by PeterBowman** [roboticslab-uc3m/kinematics-dynamics#174]
Given https://github.com/roboticslab-uc3m/teo-bimanipulation/issues/3, it would be nice to perform dual-arm control on TEO with the *keyboardController*, which currently accepts one manipulator. In that demo, coordinated coincident-TCP motion is to be achieved with two manipulators (TEO arms).

Currently WIP at https://github.com/roboticslab-uc3m/kinematics-dynamics/commit/d724a953747340c04a4334fdb90f461fd9cfce60, I'd like to add an abstraction layer so that an arbitrary number of manipulators could be controlled simultaneously.

**Comment by PeterBowman** [comment]
Might be worth splitting into a new app now that we've got a tree solver (https://github.com/roboticslab-uc3m/kinematics-dynamics/issues/190)?

**Comment by PeterBowman** [comment]
Implemented in https://github.com/roboticslab-uc3m/kinematics-dynamics/commit/8ce09b8c78acee3448bb22ac21f7d1f704223cf2 and described at https://github.com/roboticslab-uc3m/kinematics-dynamics/issues/190#issuecomment-856661951 (includes demo). Changes consists of a new parameter added to KdlTreeSolver that assumes two or more TCPs are to be treated as a single one. Neither the controller (BCC) nor the application (*keyboardController*) have been altered.

By the way, I have also deprecated `bool ICartesianSolver::getNumJoints(int&)` in favor of `int getNumJoints()` (https://github.com/roboticslab-uc3m/kinematics-dynamics/commit/bbb4cae24e0431c649890a14333f2830f33da5d7), and added `int getNumTcps()` (https://github.com/roboticslab-uc3m/kinematics-dynamics/commit/c19570aef8a0899f89addaabd2430be6f364790e).
